qiskit_ibm_runtime/options/zne_options.py

```diff
@@ -48,8 +93,25 @@ class ZneOptions:
             * `"gate_folding_back"` uses 2-qubit gate folding to amplify noise. If the noise
               factor requires amplifying only a subset of the gates, then these gates are selected
               from the back of the topologically ordered DAG circuit.
-
-        noise_factors: Noise factors to use for noise amplification. Default: (1, 3, 5).
+            * `"pea"` uses a technique called probabalistic error amplification (`PEA
```

Comment:
PEA only makes sense if gate twirling is on. Yet, `twirling.enable_gates` is `False` by default. I've forgotten about this many times while playing with PEA, and the performance wasn't great. Do we want to add a note about this, and perhaps even a warning?

Reply:
I believe it turns twirling on for you when pea is enabled. If this is not the case, I think it's a bug.

In any case, yes a note would be worthwhile.